functions.js

```diff
@@ -294,7 +294,29 @@ exports.providerMessageHandler = (req, res) => {
 
     return kp.sendMsg(req, res);
 };
+//Eksaath Callback from StepOne For Workflow stages
+exports.providerEksaathHandler = (req,res) => {
+    //Authenticate
+    if (!req.query || req.query.api_key != config.eksaath_callback.api_key.toString()) {
+        return res.status(403).send({ errorMessage: "Unauthorized Access" });
+    } else {
+        delete req.query.api_key;
+    }
+    console.log("Query: ", JSON.stringify(req.query));
+    // if (Object.keys(req.query).length === 0 && req.query.constructor === Object) {
+    //     return res.status(400).send({ errorMessage: "Eksaath icket Details required as Query Params" });
+    // }
+    var parsedUrl = getParsedUrl(req);
```

Comment:
instead of a get call, we should ideally work with a post call with json body. Makes it easier to work with that without unnecessary transformations.

Reply:
Changed this to a POST call